### PR TITLE
fix TSMReader.Delete() and all unit tests is pass in the windows

### PR DESCRIPTION
**Pull request by runner-mei**

### CODE REVIEW

**Comment by corylanou** [comment]
It looks like the build is failing on linux:

```
FAIL	github.com/influxdb/influxdb/services/continuous_querier [build failed]
```

**Comment by jwilder** [comment]
@runner-mei If you rebase w/ master, it should fix that `continuous_querier` build failure.

**Comment by runner-mei** [comment]
I rebase it.

File: tsdb/engine/tsm1/tombstone.go [hunk 1]
```diff
@@ -78,6 +79,8 @@ func (t *Tombstoner) TombstoneFiles() []FileStat {
 	return nil
 }
 
+var is_windows = "windows" == runtime.GOOS
```
Comment (corylanou) [inline tsdb/engine/tsm1/tombstone.go]: This should probably be done by creating specific operating system files and defining each method in that specific file.

**Comment by corylanou** [comment]
+1 Lookg good.  /cc @jwilder

**Comment by jwilder** [comment]
:+1:

**Comment by jwilder** [comment]
Thanks @runner-mei!

**Pull request merged**